src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -169,9 +173,10 @@ export class InputDatePicker
     }
   }
 
-  /** Expand or collapse when calendar does not have input */
   @Prop({ mutable: true, reflect: true }) active = false;
 
+  @Prop({ mutable: true, reflect: true }) open = false;
```

Comment:
I have a PR to handle this here: https://github.com/Esri/calcite-components/pull/4972

For now, just add the `open` prop as a private variable.